scripts/benchmarks_ci.py

```diff
@@ -225,6 +225,22 @@ def __is_valid_datetime(dt: str) -> str:
         action='store_true',
         help='Turns off verbosity.',
     )
+    parser.add_argument(
+        '--build-only',
+        dest='build_only',
+        required=False,
+        default=False,
+        action='store_true',
+        help='Builds the benchmarks but does not run them.',
+    )
+    parser.add_argument(
+        '--test-only',
```

Comment:
nit: replace `--test-only` with `--run-only` as run maps `dotnet run`

Reply:
I will take this comment, but I am not ready for PR right now.